packages/flutter_tools/templates/create/.analysis_options.tmpl

```diff
@@ -0,0 +1,70 @@
+# This file contains the analysis options used by the Dart analyzer
+# to decide what errors, warnings, and hints should be shown.
+# See the configuration guide for more
+# https://github.com/dart-lang/sdk/tree/master/pkg/analyzer#configuring-the-analyzer
+
+analyzer:
+  language:
+    enableStrictCallChecks: true
+    enableSuperMixins: true
+  strong-mode: false
```

Comment:
I flipped this back to "true" in https://github.com/flutter/flutter/pull/5679 because (1) that was the original value and (2) that's where Dart is moving in general.